Sources/SwiftDocC/Semantics/Symbol/DocumentationDataVariants.swift

```diff
@@ -75,6 +99,10 @@ extension DocumentationDataVariants {
         }
     }
     
+    static var none: DocumentationDataVariants<Variant> {
```

Comment:
Nit: `empty` might be more appropriate here because we're referring to a collection-like container